ci/aws/provisioner.tf

```diff
@@ -30,16 +33,33 @@ ${var.ANSIBLE_CMD} -u ${var.sudo_user} \
 ${var.SETUP_K8S_NODE} \
 --key-file ${var.private_key_file} \
 --extra-vars "\
-snaps_ci_priv_key=${var.private_key_file} \
+snaps_ci_priv_key=${var.private_key_file}
 snaps_ci_pub_key=${var.public_key_file}
 "\
 EOT
   }
 }
 
+# Call ansible script to setup K8s nodes
+resource "null_resource" "snaps-k8s-node-nic-config" {
+  depends_on = [null_resource.snaps-k8s-node-setup]
+
+  # Configure eth1 to secondary subnet
+  provisioner "local-exec" {
+// TODO - swap commands to disable the secondary NIC as it causes problems
```

Comment:
remove the commented out lines in this module?

Reply:
I left them there on purpose for the next person who works on the NIC issue